src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-function/GeneFunction.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { RootState } from 'src/store';
```

Comment:
this is a type; we generally place types just above styles

Reply:
That's done